deployment/network-operator/values.yaml

```diff
@@ -24,6 +24,11 @@ sriovNetworkOperator:
 
 # Node Feature discovery chart related values
 node-feature-discovery:
+  image:
+    repository: gcr.io/k8s-staging-nfd/node-feature-discovery
+    tag: v0.6.0-233-g3e00bfb
+  master:
+    instance: "network-operator"
```

Comment:
let remove this to be hard coded

Reply:
One we'll start using chart from NFD helm repository, we won't be able to hardcode it without network-operator and/or admission controller changes, so I)'m not sure that it's a good idea to do it